packages/radio-button/src/RadioButton.js

```diff
@@ -52,42 +47,42 @@ export default class RadioButton extends Component {
     value: PropTypes.string
   };
 
-  render() {
-    const {
-      checked: controlledChecked,
-      defaultChecked,
-      disabled,
-      label,
-      name,
-      onBlur,
-      onChange,
-      onClick,
-      onFocus,
-      required,
-      value
-    } = this.props;
+  static defaultProps = {
+    defaultChecked: false
+  };
+
+  state = {
+    checked: this.props.defaultChecked
+  };
+
+  /**
+   * @param {MouseEvent} event
+   */
+  handleChange = event => {
+    if (this.props.onChange) {
+      this.props.onChange(event);
+    }
 
+    this.setState({
+      checked: this.props.value
```

Comment:
I don't understand what's happening here. `state.checked` is a boolean, while `props.value` is a string. They're separate concerns.